src/main/proto/marathon.proto

```diff
@@ -274,6 +274,18 @@ message GroupDefinition {
   repeated string dependencies = 5;
 }
 
+message StoredGroup {
```

Comment:
You could also extend GroupDefinition, instead of creating another message.
E.g.
```protobuf
message GroupDefinition {
  message AppReference {
    required string id = 1;
    required string version = 2;
  }
  required string id = 1;
  required string version = 2;
  repeated ServiceDefinition deprecatedApps = 3;
  repeated GroupDefinition groups = 4;
  repeated string dependencies = 5;
  repeated AppReference appIds = 6;
}
```